source/core/NumericString.cpp

```diff
@@ -956,7 +909,8 @@ void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], Strin
     if (argumentIndex < count) {
         // make sure at least all of the args are used (even if out of order)
         if (usedArguments != count) {
-            ErrFormatExtraNumArgs(format, count - argumentIndex, arguments, buffer);
+            SetFormatError(kFormatTooFewFormatSpecs, count - argumentIndex, fOptions.FormatChar, errPtr);
```

Comment:
Not sure whether we would run into this. If number of arguments don't match number of format specifiers, it is a type-prop error in LabVIEW.